srcalc/src/main/java/gov/va/med/srcalc/security/VistaUserDetailsService.java

```diff
@@ -1,56 +0,0 @@
-package gov.va.med.srcalc.security;
-
-import javax.inject.Inject;
-import javax.security.auth.login.AccountException;
-
-import gov.va.med.srcalc.domain.VistaPerson;
-import gov.va.med.srcalc.vista.VistaPersonDao;
-
-import org.springframework.security.core.userdetails.*;
-
-/**
- * A Spring UserDetailsService implementation that loads users from VistA.
- */
-public class VistaUserDetailsService implements UserDetailsService
```

Comment:
No, we're still using Spring Security. We can discuss more, and I plan to add a description of our security setup to the Technical Manual.